database/migrations/2023_01_07_005110_create_mood_tracking_parameters_table.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use App\Models\Vault;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('mood_tracking_parameters', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('vault_id');
+            $table->string('label');
+            $table->string('hex_color');
+            $table->integer('position')->nullable();
+            $table->timestamps();
+            $table->foreign('vault_id')->references('id')->on('vaults')->onDelete('cascade');
+        });
+
+        Vault::chunk(200, function ($vaults) {
+            foreach ($vaults as $vault) {
+                DB::table('mood_tracking_parameters')->insert([
+                    'vault_id' => $vault->id,
+                    'label' => trans('vault.settings_mood_tracking_parameters_awesome'),
```

Comment:
Translated label should not go into the database directly...

Reply:
fixed